src/sass/components/_hero-header.scss

```diff
@@ -0,0 +1,67 @@
+.container--header-bg {
+  background-color: $pinkBg;
+}
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-top: 6px;
+  &__hamburger {
+    background-color: transparent;
+    border: none;
+    cursor: pointer;
+  }
+}
+.container {
+  overflow: clip;
+  position: relative;
+}
+.hero {
+  background-color: $pinkBg;
+  margin-top: -1px;
+  margin-bottom: -1px;
+  overflow: clip;
+  &__h1-style {
+    padding-top: 137px;
```

Comment:
W tym wypadku odsuwasz się od krawędzi rodzica, więc to rodzic powinien mieć ten padding, który wpychałby swoje dzieci do środka

Reply:
j.w.